pkg/cel/libs/context/impl.go

```diff
@@ -27,13 +27,15 @@ func (c *impl) get_configmap_string_string(args ...ref.Val) ref.Val {
 	}
 }
 
-func (c *impl) get_globalreference_string(ctx ref.Val, name ref.Val) ref.Val {
-	if self, err := utils.ConvertToNative[Context](ctx); err != nil {
+func (c *impl) get_globalreference_string(args ...ref.Val) ref.Val {
+	if self, err := utils.ConvertToNative[Context](args[0]); err != nil {
```

Comment:
we should have an arg length check here, since we are accessing elements by index